validation/vortex_street_2d/vortex_street_2d.jl

```diff
@@ -0,0 +1,236 @@
+
+else
+    state_equation = nothing
+
+    viscosity = ViscosityAdami(nu=kinematic_viscosity)
+
+    fluid_system = EntropicallyDampedSPHSystem(fluid, smoothing_kernel,
+                                               smoothing_length,
+                                               sound_speed, viscosity=viscosity,
+                                               density_calculator=fluid_density_calculator,
+                                               buffer_size=n_buffer_particles)
+end
+
+# ==========================================================================================
+# ==== Open Boundary
+
+function velocity_function2d(pos, t)
+    return SVector(prescribed_velocity, 0.0)
+end
+
+open_boundary_model = BoundaryModelTafuni()
+
+boundary_type_in = InFlow()
+plane_in = ([0.0, 0.0], [0.0, domain_size[2]])
+inflow = BoundaryZone(; plane=plane_in, plane_normal=flow_direction, open_boundary_layers,
+                      density=fluid_density, particle_spacing,
+                      boundary_type=boundary_type_in)
+
+reference_velocity_in = velocity_function2d
+reference_pressure_in = nothing
+reference_density_in = nothing
+open_boundary_in = OpenBoundarySPHSystem(inflow; fluid_system,
+                                         boundary_model=open_boundary_model,
+                                         buffer_size=n_buffer_particles,
+                                         reference_density=reference_density_in,
+                                         reference_pressure=reference_pressure_in,
+                                         reference_velocity=reference_velocity_in)
+
+boundary_type_out = OutFlow()
+plane_out = ([domain_size[1], 0.0], [domain_size[1], domain_size[2]])
+outflow = BoundaryZone(; plane=plane_out, plane_normal=(-flow_direction),
+                       open_boundary_layers, density=fluid_density, particle_spacing,
+                       boundary_type=boundary_type_out)
+
+reference_velocity_out = nothing
+reference_pressure_out = nothing
+reference_density_out = nothing
+open_boundary_out = OpenBoundarySPHSystem(outflow; fluid_system,
+                                          boundary_model=open_boundary_model,
+                                          buffer_size=n_buffer_particles,
+                                          reference_density=reference_density_out,
+                                          reference_pressure=reference_pressure_out,
+                                          reference_velocity=reference_velocity_out)
+# ==========================================================================================
+# ==== Boundary
+boundary_model = BoundaryModelDummyParticles(pipe.boundary.density, pipe.boundary.mass,
+                                             AdamiPressureExtrapolation(),
+                                             state_equation=state_equation,
+                                             viscosity=nothing, # free-slip
```

Comment:
```suggestion
```
Don't do that, or setting `viscosity=ViscosityMorris` or so with `trixi_include` will also change it here.